ERCS/erc-7758.md

```diff
@@ -0,0 +1,530 @@
+---
+eip: 7758
+title: Transfer With Authorization
+author: Peter Jihoon Kim (@petejkim), Kevin Britz (@kbrizzle), David Knott (@DavidLKnott), Dongri Jin (@dongri)
+discussions-to: https://github.com/ethereum/EIPs/issues/3010
```

Comment:
Please create a discussions topic in Eth Magicians.  It just needs to contain a link to this PR.
https://ethereum-magicians.org/

Reply:
I signed up, but it seems I can't post yet. I'll add the link once I can post.

Reply:
https://github.com/ethereum/ERCs/pull/598/commits/164054885890db83a32b1659178e0910d64a0ce3
@abcoathup DONE!